components/tyk-dashboard/templates/deployment-dashboard.yaml

```diff
@@ -155,6 +153,12 @@ spec:
             
           - name: TYK_DB_HTTPSERVEROPTIONS_USESSL
             value: "{{ .Values.global.tls.dashboard }}"
+          - name: TYK_DB_HTTPSERVEROPTIONS_MINVERSION
+            value: "771"
+          - name: TYK_DB_HTTPSERVEROPTIONS_CERTIFICATES
+            value: '[{"domain_name": "*", "cert_file": "/etc/certs/tls.crt", "key_file": "/etc/certs/tls.key"}]'
```

Comment:
What do you think if we let users define this environment variable through `extraEnvs` field by users? In my opinion, it shouldn't be a hardcoded value in the deployment and users should declare it through extraEnvs based on their certificates.

Reply:
Can you think of any other edge case in which a different setup might be needed? I cannot see one at first glance

Reply:
domain_name field might change, or users mount secrets with other names such as `server.crt` or `pem` file etc. In my opinion, it should be configurable by the user

Reply:
Agreed. I don't like how this would look if we were to add it as is. Maybe add a section then parse it to this json:
```
httpserveroptions:
- domain_name: ""
   cert_file: ""
   key_file: ""
```

Reply:
I added the following to the TLS section of each component

```
    certificatesMountPath: "/etc/certs/tyk-gateway"
    certificates:
    - domain_name: "*"
      cert_file: "/etc/certs/tyk-gateway/tls.crt"
      key_file: "/etc/certs/tyk-gateway/tls.key"
```